www/src/components/homepage/homepage-blog-posts.js

```diff
@@ -114,108 +114,96 @@ const ViewAll = () => (
   </ViewAllStyle>
 )
 
-class HomepageBlogPosts extends Component {
-  desktopMediaQuery
+const splitPostsToColumns = posts =>
+  posts.reduce(
+    (merge, post, idx) => {
+      if (idx % 2) {
+        merge[1].push(post)
+      } else {
+        merge[0].push(post)
+      }
 
-  state = {
-    desktopViewport: false,
-  }
+      return merge
+    },
+    [[], []]
+  )
 
-  componentDidMount = () => {
-    this.desktopMediaQuery = window.matchMedia(`(min-width: ${breakpoints[3]}`)
-    this.desktopMediaQuery.addListener(this.updateViewPortState)
-    this.setState({ desktopViewport: this.desktopMediaQuery.matches })
-  }
+export default function HomepageBlogPosts({ posts }) {
+  const [desktopViewport, setDesktopViewport] = useState(false)
+  let desktopMediaQuery = null
 
-  componentWillUnmount = () => {
-    this.desktopMediaQuery.removeListener(this.updateViewPortState)
-  }
+  useEffect(() => {
+    desktopMediaQuery = window.matchMedia(`(min-width: ${breakpoints[3]}`)
```

Comment:
Delete the definition above and do this:

```suggestion
    const desktopMediaQuery = window.matchMedia(`(min-width: ${breakpoints[3]}`)
```